### ISSUE TITLE
Firefox - Unable to route 

### ISSUE THREAD

**Issue by MonolithNZ** [gyng/save-in#196]
Hi there,

Thanks very much for the amazing extension. I'm trying to route .STL files into /downloads/STL , but I'm having no luck. 

fileext: stl 
into: STL

I've tried a few basic variations of the above. What am I doing wrong here?

Thanks again

**Comment by MonolithNZ** [comment]
Yeah I gave up on this. If you find an alternative way to auto organize downloads let me know. It's possible with extensions in Chrome, can't believe it's been such a struggle to find something that provides this functionality with FF

**Comment by akhiljalagam** [comment]
doesnt work for me.

**Comment by C-BoT-AU** [comment]
Same basic needs/wants to route from site to folder within downloads. Nothing seems to work.

> Tried https://addons.mozilla.org/en-US/firefox/addon/quick-sorted-download/reviews/ But it didn't work at all. Might be because i'm on Waterfox, perhaps worth a try if you're on regular FF. It's regex.

No joy. Running Firefox so I'm wondering if at some point during the past few updates something was changed that blocks this sort of extension from working.

**Comment by Juts2** [comment]
Agreed, this doesnt work any longer
